pages/common/zoxide.md

```diff
@@ -0,0 +1,28 @@
+# zoxide
+
+> Keep track of the directories you use most frequently, and use a ranking algorithm to navigate to the best match.
```

Comment:
Perhaps this will read better?
```suggestion
> Keep track of the most frequently used directories.
> Uses a ranking algorithm to navigate to the best match.
```